src/App.vue

```diff
@@ -225,9 +227,12 @@ export default class App extends Mixins(
 
     // Set base crumbs based on user role
     // Staff don't want the home landing page and they can't access the Manage Business Dashboard
-    if (this.isRoleStaff) {
-      // If staff, set StaffDashboard as home crumb
-      crumbs.unshift(getStaffDashboardBreadcrumb())
+    if (this.isAuthenticated) {
+      if (this.isRoleStaff) {
+        crumbs.unshift(getStaffDashboardBreadcrumb())
+      } else {
+        crumbs.unshift(getUserDashboardBreadcrumb())
+      }
     } else {
       // For non-staff, set Home crumb
```

Comment:
"For non-staff, or not logged in, set Registry Dashboard Breadcrumb"

Reply:
Please update comment. And please rename the variable to `getRegistryHomeBreadcrumb`.